docs/tutorial/part_4.rst

```diff
@@ -142,6 +142,22 @@ suites for end-to-end tests (``StaticLiveServerTestCase`` or ``LiveServerTestCas
 that URLs inside the Channels routing configuration like ``/ws/room/ROOM_NAME/``
 will work inside the suite.
 
+We are using ``sqlite3``, which for testing, is treated as a database in memory, and therefore, the tests will not run correctly.
+We need to tell our project that the ``sqlite3`` database need not to be in memory for run the tests. Edit the
+``mysite/settings.py`` file and add ``TEST`` argument to the **DATABASES** setting::
```

Comment:
"add test argument" -> "add the test argument"

Reply:
Thanks again!